src/Event/SmsEvents.php

```diff
@@ -36,12 +36,81 @@
   const MESSAGE_PRE_PROCESS = 'sms.message.pre_process';
 
   /**
-   * Process and chunk a SMS message after it is queued, sent, or received.
+   * Process an SMS message after it is sent or received.
```

Comment:
Comment should be more like "Event that is dispatched after an SMS message is sent or received."